content/en/data_streams/setup/technologies/kafka.md

```diff
@@ -56,6 +56,19 @@ title: Data Streams Monitoring for Kafka
 
 <div class="alert alert-info"><a href="https://kafka.apache.org/documentation/streams/">Kafka Streams</a> is partially supported for Java, and can lead to latency measurements being missed.</div>
 
+
+### Supported Kafka Deployments
+
+Instrumenting your consumers and producers with Data Streams Monitoring allows you to view your topology and track your pipelines with [ready to go metrics][7] independently of how Kafka is deployed. However, the following Kafka deployments have further integration support, providing additional insights into the health of your Kafka cluster.
```

Comment:
I thought "However" made it sound like there was more of a contradiction at play than there actually was, but YMMV.
```suggestion
Instrumenting your consumers and producers with Data Streams Monitoring allows you to view your topology and track your pipelines with [ready-to-go metrics][7] independently of how Kafka is deployed. Additionally, the following Kafka deployments have further integration support, providing more insights into the health of your Kafka cluster:
```